CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## [1.2.12](https://github.com/ably/ably-cocoa/tree/1.2.12)
+
+[Full Changelog](https://github.com/ably/ably-cocoa/compare/1.2.11...1.2.12)
+
+**Fixed bugs:**
+
+- SCNetworkReachabilitySetCallback (Crashed: io.ably.main) [\#1380](https://github.com/ably/ably-cocoa/issues/1380), fixed in: Fix of the crash in the `ARTOSReachability` [\#1453](https://github.com/ably/ably-cocoa/pull/1453) ([maratal](https://github.com/maratal))
+
```

Comment:
You've got an empty line here which shows up in the rendered markdown (on github at least) and isn't consistent with the rest of the changelog. I'll approve this PR anyway since everything else looks fine and I don't wanna block this, but you should fix this before merging.

Reply:
https://github.com/ably/ably-cocoa/pull/1454/commits/125ba2b93a602080932751459ccad0d451319f3c